File: kattis/ASCII_kassi/ASCII_kassi.c
#include <stdio.h>
#include <cs50.h>

int main() {
    int n;
    int i;
    int j;
    n = get_int("");

    // Top square
    printf("+");
    for(i = 0; i < n; i++) {
        printf("-");
    }
    printf("+ \n");

    // Middle square
    for(i = 0; i < n; i++) {
        printf("|");

        for(j = 0; j < n; j++) {
            printf(" ");
        }

        printf("|\n");
    }

    // bottom square
    printf("+");
    for(i = 0; i < n; i++) {
        printf("-");
    }
    printf("+ \n");
}

File: kattis/ASCII_kassi/ASCII_kassi.py
# ASCII kassi

"""
Draw a square using 
    |, -, +
left and right sides are |, top and bottom are -
corners are +
"""

n = int(input())

def draw_square(n):
    top_bottom = '-' * n
    print('+' + top_bottom + '+')
    # print vertical
    if n > 0:
        for i in range(0,n):
            print('|' + ' ' * n + '|')    
    print('+' + top_bottom + '+')

draw_square(n)

